docs/schema.md

```diff
@@ -0,0 +1,38 @@
+# AWS SAM JSON schema
+
+## Context
+
+The [AWS SAM specification](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification.html) is defined by a [JSON schema](https://github.com/aws/serverless-application-model/discussions/2645).
```

Comment:
Very non-blocking nit: I think the schama is derived from the SAM code, right? So technically, it isn't defined by the schema.

Reply:
Any suggestions on wording?

Reply:
The @acristin 's "derived" is good. A json schema is derived from SAM spec and xxx